docs/modules/eslint-plugin-react.md

```diff
@@ -6,7 +6,53 @@
 
 ## `@eslint-react/eslint-plugin`
 
-Feature rich alternative with many of the same rules.
+Not a drop-in replacement but a feature rich alternative with many of the same rules.
 
 [Project Page](https://github.com/Rel1cx/eslint-react)
 [npm](https://www.npmjs.com/package/@eslint-react/eslint-plugin)
+
+## `@stylistic/eslint-plugin-jsx`
+
+The following stylistic rules in `eslint-plugin-react` can be replaced by the same rules in `@stylistic/eslint-plugin-jsx`:
+
+- `react/jsx-child-element-spacing`
+- `react/jsx-closing-bracket-location`
+- `react/jsx-closing-tag-location`
+- `react/jsx-curly-brace-presence`
+- `react/jsx-curly-newline`
+- `react/jsx-curly-spacing`
+- `react/jsx-equals-spacing`
+- `react/jsx-first-prop-new-line`
+- `react/jsx-indent-props`
+- `react/jsx-indent`
+- `react/jsx-max-props-per-line`
+- `react/jsx-newline`
+- `react/jsx-one-expression-per-line`
+- `react/jsx-pascal-case`
+- `react/jsx-props-no-multi-spaces`
+- `react/jsx-sort-default-props`
+- `react/jsx-sort-props`
+- `react/jsx-space-before-closing`
+- `react/jsx-tag-spacing`
+- `react/jsx-wrap-multilines`
+
+[Project Page](https://eslint.style/packages/jsx)
+[npm](https://www.npmjs.com/package/@stylistic/eslint-plugin-jsx)
+
+## TypeScript
```

Comment:
im not sure if we should bother listing this typescript section. as we don't really explain what replaces them or how (and that doesn't belong in these docs. rather it'd be better once we start work on our extended docs)

Reply:
> im not sure if we should bother listing this typescript section. as we don't really explain what replaces them or how (and that doesn't belong in these docs. rather it'd be better once we start work on our extended docs)

Good point. I'll remove the TypeScript section for now.